src/gluonts/torch/model/gnninf/module.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# If you use this code in your work please cite:
+# Multivariate Time Series Forecasting with Latent Graph Inference
+# (https://arxiv.org/abs/2203.03423)
+
+
+import torch
+from torch import nn
+from gluonts.torch.modules.distribution_output import StudentTOutput
+from typing import Callable
+from .networks.model import GNNInfNetwork
+from typing import Tuple, Any
+
+
+def mean_abs_scaling(
+    context: torch.Tensor, min_scale: float = 1e-3
+) -> torch.Tensor:
+    return context.abs().mean(1).clamp(min_scale, None).unsqueeze(1)
+
+
+class GNNInfModule(nn.Module):
```

Comment:
I think it would be nice to document at least also this class, since it appears like the main entry point for the model in case someone wants to interact with it directly (that is, not through the estimator class)